visualization/app/codeCharta/core/data/data.service.spec.ts

```diff
@@ -70,8 +70,8 @@ describe("app.codeCharta.core.data.dataService", function() {
 
     it("set metrics should set metrics correctly with multiple maps", ()=>{
         dataService.setMap(data, 0);
-        let data2 = JSON.parse(JSON.stringify(data));
-        data2.root.children[0].attributes["test"] = 0;
+        let data2: CodeMap = JSON.parse(JSON.stringify(data));
```

Comment:
Since data2 has a type now, we no longer need to unnecessary stringify->parse the object to give it an any-type.

Reply:
resolved: used for copying